test/Market.t.sol

```diff
@@ -1842,6 +1850,7 @@ contract Market_validates_certificate_amount is UpgradeableUSDCMarket {
 
   function test() external {
     uint256 ownerPrivateKey = 0xA11CE;
+    uint256 priceMultiple = _market.getPriceMultiple();
```

Comment:
this has to be set here as a variable otherwise the  `expectRevert` gets applied to the `getPriceMultiple` call as it is the next call that occurs

Reply:
probably belongs in the setup function anyways